mooncake-store/src/master_service.cpp

```diff
@@ -474,7 +481,8 @@ void MasterService::GCThreadFunc() {
             MasterMetricManager::instance().dec_key_count(gc_count);
         }
 
-        if (need_eviction_ && eviction_ratio_ > 0.0) {
+        if (MasterMetricManager::instance().get_global_used_ratio() > eviction_high_water_mark_ratio_
```

Comment:
Revoking BatchEvict only once does not guarantee that global_used_ratio remains below high_watermark_ratio?

Reply:
@stmatengss Yeah, it based on the next loop eviction, I push a new commit to improve this.